waiter/specs/k8s-default-pod.edn

```diff
@@ -0,0 +1,39 @@
+{:kind "ReplicaSet"
```

Comment:
Let's move this into the code.  It will simplify customizations to the `ReplicaSet`.